search-app/src/cmr/search/services/query_execution/facets/collection_v2_facets.clj

```diff
@@ -26,7 +26,8 @@
    :project :project-sn-humanized
    :processing-level-id :processing-level-id-humanized
    :variables :variables
-   :granule-data-format :granule-data-format-humanized})
+   :granule-data-format :granule-data-format-humanized
+   :coordinate-system :two-d-coordinate-system-name})
```

Comment:
This facet is going to be named `tiling-system` per discussion with @eereiter and the EDSC team